src/hugolib/aasa_test.go

```diff
@@ -60,11 +60,17 @@ func TestAASAOutput(t *testing.T) {
 			depsCfg := deps.DepsCfg{Fs: fs, Cfg: cfg}
 
 			writeSourcesToSource(t, "content", fs, weightedSources...)
+
+			if tv == 9 {
+				// expect the site not to build
```

Comment:
@felicianotech - how can I test that `buildSingleSite(t, depsCfg, BuildCfg{})` fails or contains "* not a valid AASA version"?

Reply:
I don't know. It's a function. Follow it's return, the type, and what that struct and its methods do. I assume somewhere in there you'll find the answer to the question.